pkg/ddc/alluxio/worker.go

```diff
@@ -26,7 +26,9 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
-// setup the cache worker
+// SetupWorkers checks current replicas at the runtime and make an update
+// over the status by setting current and scheduled phases. The function
+// calls for a status update and finally returns the given response.
```

Comment:
I think it's better to:
```suggestion
// SetupWorkers checks the desired and current replicas of workers and makes an update
// over the status by setting phases and conditions. The function
// calls for a status update and finally returns error if anything unexpected happens.
```